src/main/app.js

```diff
@@ -100,7 +100,8 @@ const isSecondInstance = app.makeSingleInstance(argv => {
 });
 
 if (isSecondInstance) {
-  app.quit();
+  log.info('Second instance opened, attempting to exit now.');
+  app.exit();
```

Comment:
What is the behaviour of `isSecondInstance` in this case:

 1. Version 10.0.0 is running
 2. Version 10.1.0 is started

Does version 10.0.0 close? Because I'd imagine it should, especially in the case of using AppImage builds..

Reply:
Second instance is just basically the second running instance regardless of version. But I imagine we can configure the behavior by manually checking the versions and alerting the user or something? But if so, lets create an issue for it and fix it separately

Reply:
> regardless of version

Ok, so that means that electron doesn't know which is the "second instance"? Unfortunately it might be quite critical: If the user is trying to open a new instance, and they cannot due to it being closed constantly, then usability isn't so great..

Wouldn't it be better to force-close the first instance, and always keep the _newly opened instance_ active?

Reply:
To move this forward, I created #597 to take care of the execution/closing.